src/main/scala/chess/format/Forsyth.scala

```diff
@@ -73,7 +73,14 @@ object Forsyth {
               lastTurn = lastTurn,
               positionHashes = Array.empty,
               castles = castles,
-              unmovedRooks = UnmovedRooks(unmovedRooks)
+              unmovedRooks = UnmovedRooks(unmovedRooks),
+              currentTurn = splitted
+                .filter(_.startsWith("½"))
+                .map(_.replace("½", ""))
+                .lift(0)
+                .flatMap(format.Uci.apply)
+                .map(List(_))
+                .getOrElse(List())
```

Comment:
This is how we should be handling the 'half move' when loading a fen in -> put it into the `situation.board.history.currentTurn`. This is how Amazons should do it too.

Reply:
I'm not sure if amazons should do it, because the board state we use in the fen for amazons is the board before the move was applied, while the board state for monsters is the move after the currentTurn move has been applied.

Perhaps I could get amazons to be the same, but then we'd have two different board states for the fen, the strategy games one and the fairysf one.

Would you like me to do that?